plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -583,8 +611,13 @@ $.extend(DataTable.prototype, UIControl.prototype, {
 
             // on resize of the window we re-calculate everything.
             var timeout = null;
+            var windowWidth = 0;
             var resizeDataTable = function() {
 
+                if (windowWidth === $(window).width()) {
```

Comment:
The window resize event is triggered if the height of the window changes. That happens e.g. if a subtable is loaded and the content height increases. As that won't affect the needed width of a column there is no need to resize them.

This might have actually be the reason for some of the resizing bugs we had when loading a subtable. As in some cases the resize for all datatables on the page was triggered.